hawc/apps/common/models.py

```diff
@@ -277,8 +277,11 @@ def create_tag(cls, assessment_id, parent_id=None, **kwargs):
         else:
             parent = cls.get_assessment_root(assessment_id)
 
+        # TODO make this a regex check to prevent adding a name that blocks another assessment
```

Comment:
is that needed? we could just update the queryset to only check for a depth of 1, which users couldn't edit I think

Reply:
Good call! I included that in the newest PR